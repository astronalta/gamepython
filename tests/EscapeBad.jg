

main() Int {
    "\x"
    "\o"
    "\x1"
    "\xg"
    "\o1"
    "\o11"
    "\o11f"
    '\x'
    '\o'
    '\x1'
    '\xg'
    '\o1'
    '\o11'
    '\o11f'
    ret 0 
}
