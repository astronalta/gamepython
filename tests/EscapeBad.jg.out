EscapeBad.jg:4:5: Invalid hexadecimal sequence in literal
EscapeBad.jg:5:5: Invalid octal sequence in literal
EscapeBad.jg:6:5: Invalid hexadecimal sequence in literal
EscapeBad.jg:7:5: Invalid hexadecimal sequence in literal
EscapeBad.jg:8:5: Invalid octal sequence in literal
EscapeBad.jg:9:5: Invalid octal sequence in literal
EscapeBad.jg:10:5: Invalid octal sequence in literal
EscapeBad.jg:11:5: Invalid hexadecimal sequence in literal
EscapeBad.jg:12:5: Invalid octal sequence in literal
EscapeBad.jg:13:5: Invalid hexadecimal sequence in literal
EscapeBad.jg:14:5: Invalid hexadecimal sequence in literal
EscapeBad.jg:15:5: Invalid octal sequence in literal
EscapeBad.jg:16:5: Invalid octal sequence in literal
EscapeBad.jg:17:5: Invalid octal sequence in literal
